ui/src/app/shared/pipes/capitalize-filter.pipe.ts

```diff
@@ -0,0 +1,16 @@
+import { PipeTransform, Pipe } from '@angular/core';
+
+/**
+ * Pipe that transforms an input string to camel-case.
+ * 
+ * @author Gunnar Hillert
+ */
+@Pipe({name: 'capitalize'})
+export class CapitalizePipe implements PipeTransform {
+  transform(input: string): string{
```

Comment:
Curious why we named the .ts file capitalize-filter.pipe.ts.   Its more of a transform.